Makefile.am

```diff
@@ -180,10 +175,8 @@ bumpver: po-pull
 	if [ ! -z "$(SKIP_ACKS)" ]; then \
 		opts="$${opts} -s" ; \
 	fi ; \
-	( cd $(srcdir) && scripts/makebumpver $${opts} ) || exit 1 ; \
-	$(MAKE) -C po $(PACKAGE_NAME).pot-update && \
-	rm $(srcdir)/po/{main,extra}.pot
-	$(ZANATA_CLIENT_BIN) push $(ZANATA_PUSH_ARGS)
+	( cd $(srcdir) && scripts/makebumpver $${opts} ) || exit 1 ;
```

Comment:
I wonder if there should bre '\' at the and of the line. Perhaps not necessary, but there are places we do that, so I better ask.

Reply:
It's correct. We don't share anything from the above in the last line so there is no reason to have it as one shell.
However, I removed have `;` on the last line. :)